src/components/PrintableTabContent/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+type PrintableTabContentProps = {
+  visible: boolean;
+  children: React.Node;
+};
+
+const PrintableTabContent = ({
```

Comment:
It's possible that this component needs a better name, since it's not really specific to tabs now that it has been extracted.